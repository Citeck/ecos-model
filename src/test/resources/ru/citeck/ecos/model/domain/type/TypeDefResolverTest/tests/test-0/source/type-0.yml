---
id: type-0

formRef: uiserv/form@form-0

properties:
  type-0-prop: value-0
