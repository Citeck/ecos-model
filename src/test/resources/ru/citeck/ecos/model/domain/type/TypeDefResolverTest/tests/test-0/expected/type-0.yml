---
id: type-0
name:
  en: type-0

formRef: uiserv/form@form-0
parentRef: emodel/type@base
inheritCreateVariants: false

properties:
  type-0-prop: value-0
